system-dependencies.sh

```diff
@@ -78,7 +78,7 @@ while ! test -z $1; do
 			unset IGNORE_HOMEBREW
 			PROVISION_SHARPIE=1
 			unset IGNORE_SHARPIE
-			PROVISION_SIMULATORS=1
+			#PROVISION_SIMULATORS=1
```

Comment:
Can we file an issue so we remember to turn this back on?

Reply:
done: https://github.com/xamarin/xamarin-macios/issues/6205